api/v1beta1/foundationdbcluster_types.go

```diff
@@ -3099,6 +3104,11 @@ func (cluster *FoundationDBCluster) GetIgnorePendingPodsDuration() time.Duration
 	return cluster.Spec.AutomationOptions.IgnorePendingPodsDuration
 }
 
+// GetIgnoreTerminatingPodsDuration returns the value of IgnoreTerminatingPodsDuration or defaults to 10 minutes.
+func (cluster *FoundationDBCluster) GetIgnoreTerminatingPodsDuration() int {
+	return pointer.IntDeref(cluster.Spec.AutomationOptions.IgnoreTerminatingPodsDuration, int(10*time.Minute.Nanoseconds()))
```

Comment:
Is there a reason we use nanoseconds here?

Reply:
I misinterpreted your previous comment and again added nanosecond here, will make the change.